### PR TITLE
Editing

### PR DESCRIPTION
**Pull request by felixlinker**
Hi Brian, Hi Tommy,

Before I form an opinion on the charter, I wanted to clarify some language. I do not intend to change the meaning of the charter with this PR, but to clarify some aspects. If you find the PR changes the meaning, please close it without merging!

In particular, I'm curious about line 23. Do you mean "cannot" or "must not be able to"? This is a critical difference in my eyes.

Cheers,
Felix

### CODE REVIEW

**Review by OR13 — COMMENTED** [review]
these look good to me

**Comment by OR13** [comment]
FYI, these changes are not present in https://datatracker.ietf.org/doc/charter-ietf-diem/00-00/ and a new version will need to be uploaded if this PR is merged

**Comment by rohanmahy** [comment]
Hi Felix,

Could you please resolve the merge conflicts on the PR?

> In particular, I'm curious about line 23. Do you mean "cannot" or "must not be able to"? This is a critical difference in my eyes.

Can you give an example of where these two things would result in different behavior? To me "cannot" in a protocol requirement is the same as not being able to.

Thanks,
-r

**Comment by felixlinker** [comment]
Hey Rohan,

Fixed the merge conflict, thanks for pointing that out!

> To me "cannot" in a protocol requirement is the same as not being able to.

Yes, absolutely. I think the problem is that this sentence was intended to be phrased as a requirement, but is phrased as a limit on the scope.

An example: "We will provide internet connectivity to devices that cannot connect to the internet." vs "We will provide internet connectivity to devices that must not be able to connect to the internet." Former makes sense, latter is contradictory.

**Comment by bkhabs** [comment]
Line 25 was intended to be a limit on the initial scope of the WG and not a requirement. That seemed to be the consensus coming out of the BoF in Dublin.

File: diem-unified-charter.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ drivers in the development of a digital emblem architecture.
 To simplify the beginning work for DIEM, the initial scope is limited
 to attested (e.g., an emblem issuer is independent of the asset, but is attesting to
 the asset's right to claim protection) and self-signed (e.g., an emblem issuer is the
-entity controlling the protected asset) emblem types and digital assets that cannot determine if
+entity controlling the protected asset) emblem types and digital assets that must not be able to determine if
```
Comment (OR13) [inline diem-unified-charter.md]:
```suggestion
entity controlling the protected asset) emblem types and digital assets that cannot determine if
```

**Pull request merged**